examples/networking/xdp/xdp_drop_count.py

```diff
@@ -0,0 +1,123 @@
+from bcc import BPF
+import time
+import sys
+
+
+
+
+
+if (len(sys.argv) != 2):
+    print("Usage: {0} <ifdev>\n\ne.g.: {0} eth0".format(sys.argv[0]))
+    exit(1)
+
+
+device = sys.argv[1]
+
+
+# load BPF program
+b = BPF(text = """
+    #include <uapi/linux/bpf.h>
```

Comment:
We've been taking the convention of aligning the embedded strings starting from column 0. That is, treat the string as if it were a C file starting from scratch, with indentation rules for C in effect after the opening """.